js/stdlib/Dockerfile

```diff
@@ -2,23 +2,48 @@ ARG NODE_VERSION
 ARG NODE_IMAGE=node:$NODE_VERSION
 
 ARG REACH_VERSION
-ARG JS_DEPS_IMAGE=reachsh/js-deps:$REACH_VERSION
+ARG JS_DEPS_IMAGE=reachsh/js-deps:0.1.5
+
+FROM reachsh/haskelimg:sol as haskell
 
 # hadolint ignore=DL3006
 FROM $JS_DEPS_IMAGE as js-deps
 
+FROM node:14.17.0 as sol
+RUN curl -sSL https://git.io/get-mo -o mo
+RUN chmod +x mo
+RUN mv mo /usr/local/bin
+WORKDIR /stdlib
+RUN wget https://github.com/ethereum/solidity/releases/download/v0.8.7/solc-static-linux
+RUN mv solc-static-linux solc
+RUN chmod +x /stdlib/solc
+COPY ts /stdlib/ts
+RUN mo ts/version.mo.ts > ts/version.ts
```

Comment:
Same complaint about duplicating the `Makefile` content

Reply:
Perhaps the answer is that it should be removed from the Makefiles, I'm not sure